static/src/stylesheets/module/content/_recipe-article.scss

```diff
@@ -426,3 +441,34 @@ $padding-wide: 100px;
         color: $purple;
     }
 }
+
+.ad-wrapper {
+  background-color: $grey-two;
+  margin-left: -20px;
+  margin-top: 20px;
+  padding-top: 6px;
+  padding-bottom: 6px;
+
+  .ad-container{
+    margin-left: auto;
+    margin-right: auto;
+    width: max-content;
+  }
+}
+
+
+.ad-container--mobile {
```

Comment:
You could use utility classes instead, i.e. `hide-until-tablet` and `mobile-only`, they're doing the same thing. Worth keeping that in mind for an upcoming version.